package.json

```diff
@@ -49,5 +49,6 @@
     "gis",
     "webgl",
     "svg"
-  ]
+  ],
+  "main": "geo.js"
```

Comment:
It allows someone to use geojs with webpack or browserify more easily without having to provide an explicit [resolver](https://webpack.github.io/docs/configuration.html#resolve).